docs/how-to/monitor-a-router/monitor-router-performance.md

```diff
@@ -0,0 +1,15 @@
+# Monitoring Router Performance
+
+{% hint style="warning" %}
+This document assumes that an appropriate monitoring solution has been integrated to the deployment of Turing.
+The actual functionality may be different, according to the deployment configurations.
+{% endhint %}
+
+1. Navigate to the Router Details View of your router.
+
+2. Click on the monitoring link of your router under Router Actions.
+![monitoring_tab](../../.gitbook/assets/monitoring_tab.png)
+
+3. This link will bring you to the Lens dashboard with standard router metrics such as 
```

Comment:
I think we should not mention `Lens` here. 😅

Reply:
Done. Thanks for catching!